rosetta/src/main/java/com/ahmedjazzar/rosetta/ProcessPhoenix.java

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Can't we just include it instead of copying and pasting it?

Reply:
i did it but was Creating problems that's why added it as a file, i tried to solve it once i will check then will commit this code.

Reply:
i did it so buddy. i added it as Dependency but for that i have to Add 
<category android:name="android.intent.category.DEFAULT" />
in my Manifest in Main Default launcher Activity, other wise was causing crash.